packages/input/src/Input/Input.tsx

```diff
@@ -0,0 +1,157 @@
+import { Slot as RadixSlot } from "@radix-ui/react-slot";
+import { useComposedRefs } from "@telegraph/compose-refs";
+import clsx from "clsx";
+import React from "react";
+
+import { COLOR, SIZE } from "./Input.constants";
+
+type BaseRootProps = {
+  size?: keyof typeof SIZE.Container;
+  variant?: keyof typeof COLOR.Container.default;
+  hasError?: boolean;
```

Comment:
I know this is an ultra naming nit 🫣 and feel free to ignore - but slight preference toward `errored` or `isErrored`, to be in line with `disabled`. Maybe `isErrored` better to distinguish the native input attribute vs the component one.